great_expectations/core/usage_statistics/schemas.py

```diff
@@ -827,6 +827,17 @@
                 "event_payload": {"$ref": "#/definitions/empty_payload"},
             },
         },
+        {
+            "type": "object",
+            "properties": {
+                "event": {
+                    "enum": [
+                        "expectation_suite.add_expectation",
```

Comment:
👍

Reply:
I think that this event can be bundled with other events in the schema, which use the empty payload.

Reply:
Good catch @alexsherstinsky - that makes sense. So `expectation_suite.add_expectation` would be added to the enum list that currently includes `"data_context.build_data_docs",
                        "data_context.open_data_docs",
                        "data_context.run_checkpoint",`

Reply:
Done!